aries-site/src/pages/foundation/typography.mdx

```diff
@@ -67,10 +112,10 @@ Below is an example of how Heading levels should be implemented for a page calle
 
 ```
 
-When implementing heading levels, always use the correct semantic heading level. This maintains a well structured HTML document, needed by assistive technologies.
-
 ## Paragraph
 
+Paragraphs represent blocks of flowing text and are styled using [Text design tokens](/design-tokens/typography-system#text) for font size, weight, and line height. To ensure optimal readability, Paragraphs differ from standard text by enforcing a maximum width to limit line length. Implementations should reference the corresponding Text t-shirt size tokens for typography properties and apply the associated `maxWidth` value to the container.
```

Comment:
"apply the associated `maxWidth` value to the container" <-- this technique is too particular to a specific tool.

```suggestion
Paragraphs represent blocks of flowing text and are styled using [Text design tokens](/design-tokens/typography-system#text) for font size, weight, and line height. Paragraphs differ from standard text by providing a maximum width [limiting line length for optimal readability](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-visual-presentation.html). Implementations should reference the corresponding Text t-shirt sizes and maxWidth design tokens.
```